loops.js

```diff
@@ -87,5 +111,25 @@
 // etc.
 // ==========================================
 
+for (let i = 0; i < 101; i++) {
+
+
+if (i % 3 ===0 && i % 5 !==0) {
+    console.log("Fizz");
+}
+
+
+if (i % 5 ===0 && i % 3 !==0) {
+    console.log("Buzz");
+}
+
+
+if (i % 3 ===0 && i % 5 ===0) {
+    console.log("Fizzbuzz");
+}
+if (i % 3 !==0 && i % 5 !==0 && i % 15 !==0) {
+    console.log(i);
 
+}
+}
```

Comment:
Als je "else if" statements gebruikt kan de de condities in het if statement wat versimpelen dat maakt de code wat leesbaarder en is de kans op fouten ook wat minder